products/ocp4/profiles/moderate.profile

```diff
@@ -42,6 +42,7 @@ filter_rules: '"ocp4-node" not in platforms and "ocp4-master-node" not in platfo
 # CM-6 CONFIGURATION SETTINGS
 # CM-6(1) CONFIGURATION SETTINGS | AUTOMATED CENTRAL MANAGEMENT / APPLICATION / VERIFICATION
 extends: cis
+extends: moderate-rev-4
 
 selections:
```

Comment:
Remove these selections since they're going to be inherited from the extended profile.